docs/topics/minishift-install-nexus-setup.adoc

```diff
@@ -121,55 +124,28 @@ Nexus comes pre-configured for the Maven Central Repository, but you may need ot
 oc project {project-name}
 --
 +
-. Assign the `admin` role to your Jenkins application and the `edit` permissions to your OpenShift account.
-Skip this step if you have already assigned these permissions.
-+
-[source,bash,subs="attributes+"]
---
-oc policy add-role-to-user edit system:serviceaccount:{project-name}:default
-oc policy add-role-to-user admin system:serviceaccount:{project-name}:jenkins
---
-+
-. Obtain the Jenkins server URL and use it to access the Jenkins instance from your browser.
+. Define and start a new build. Pass in a parameter to set the value of the `MAVEN_MIRROR_URL` to match the route of your Nexus application:
 +
-[source,bash,subs="attributes+"]
+[NOTE]
 --
-oc get routes jenkins
+Ensure that the YAML template of the builder image you are using for your application has the `MAVEN_MIRROR_URL` environmental variable defined.
+If it does not, see the Nexus documentation for instructions on link:https://help.sonatype.com/display/NXRM2/Maven+and+Other+Build+Tools[configuring your build manually^] before proceeding.
 --
 +
 [source,bash,subs="attributes+"]
 --
-NAME      HOST/PORT                                         PATH      SERVICES   PORT      TERMINATION
-jenkins   jenkins-{app-name}-{project-name}.{osl-route-hostname}             jenkins    <all>     edge/Redirect
+$ oc new-build {app-name}:latest~{scm-repo-url} \
+-e MAVEN_MIRROR_URL='http://nexus.{nexus-project-name}:8081/nexus/content/groups/public'
 --
 +
-. Navigate to `jenkins-{app-name}-{project-name}.{osl-route-hostname}/configure`, and log in using your OpenShift credentials to access the Jenkins configuration page.
-+
-. Go to the section titled _Kubernetes Pod Template_,  click on _Add Evironmental Variable_, and select _Global Environmental Variable_ option from the drop-down menu.
-+
-. Define a new environment variable:
-* Name: `MAVEN_MIRROR_URL`
-* Value: `nexus-{nexus-project-name}.{osl-route-hostname}/nexus/content/groups/public/`
-+
-[NOTE]
---
-The actual value of the `MAVEN_MIRROR_URL` environmental variable corresponds to the service route of the pod running the Nexus server instance on your {OpenShiftLocal}.
+. To ensure that your new build is using Nexus to retrieve artifacts, you can:
 
-To obtain this route, navigate to the project that contains your Nexus instance and execute the command `oc get routes nexus`.
---
-+
-. Click _Apply_ to confirm the configuration changes.
-+
-. Redeploy the Jenkins server instance with the latest configuration changes.
+* Navigate to `http://nexus-{nexus-project-name}.{osl-route-hostname}:8081/nexus/content/groups/public` to view the list of artifacts stored in your local repository.
```

Comment:
But this won't work actually. I think. The Nexus _service_ exposes port `8081` (and that's what is used inside the cluser), but the Nexus _route_ (which is the only thing reachable from outside) exposes port `80` as usual. Do you have this rendered somewhere, so I can be sure I read this correctly?

Reply:
Unfortunately, I not have a link to a rendered preview, but I've built the guide locally and uploaded the [HTML File](file.brq.redhat.com/home/brq/ssitani/public_html/minshift-installation.html)

Reply:
OK, thanks. I'll write a more detailed comment.